.travis.yml

```diff
@@ -75,6 +75,10 @@ jobs:
         - make go-test
         - make mod-check
 
+    - name: Build multi-architecture image for amd64 and arm64
+      script:
+        - travis_wait ./build-multi-arch-image.sh || travis_terminate 1;
```

Comment:
This might need to become part of the `deploy` section at the end of this file?

Reply:
No this is once to do build testing so that we don't regress again

Reply:
Also see #965 that includes a bit of restructuring of this file.

Reply:
yeah, forgot to add travis_wait for deploy time. done now

Reply:
On line 104 of this file, there is a special job for aarch64 builds. Should that be removed now?

Reply:
that it there actually to test cephcsi E2E on kubernetes environment. for arm64

Reply:
Yes, true. And because there are different Dockerfiles for development/local builds and `deploy/cephcsi/image/Dockerfile`, both should probably stay.